README.md

```diff
@@ -65,7 +65,7 @@ The following table outlines which clients have produced which duties on a publi
 
 | Duty \ Client                        |                      Teku                       |                    Lighthouse                     | Lodestar | Nimbus | Vouch | Prysm |
 | ------------------------------------ | :---------------------------------------------: | :-----------------------------------------------: | :------: | :----: | :---: | :---: |
-| _Attestation_                        |                       ✅                        |                        ✅                         |    🚧    |   🚧   |  🚧   |  🚧   |
+| _Attestation_                        |                       ✅                        |                        ✅                         |    ✅    |   🚧   |  🚧   |  🚧   |
 | _Attestation Aggregation_            |                       🚧                        |                        🚧                         |    🚧    |   🚧   |  🚧   |  🚧   |
 | _Block Proposal_                     |                       ✅                        |                        ✅                         |    🚧    |   🚧   |  🚧   |  🚧   |
```

Comment:
haven't updated block proposal part since it isn't tested with lodestar.
if it works on simnet can we mark it as ✅ ?

Reply:
yeah we should decide whether we are going to add all these clients to the compose tests, we should probably.

Reply:
sure, i'll add lodestar to compose tests alongside lighthouse, teku and vmock in another PR.